quickshell/Common/SettingsData.qml

```diff
@@ -100,6 +100,11 @@ Singleton {
     property bool controlCenterShowNetworkIcon: true
     property bool controlCenterShowBluetoothIcon: true
     property bool controlCenterShowAudioIcon: true
+    property bool showPrivacyButton: true
```

Comment:
I think this looks mostly okay, I would prefer to keep the current behavior though. Because when you add it to the bar, you'll see the different options you can configure. But for people who already have it there, they would get a different behavior than before and have to go into the settings to find it.